x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/api/rules/update_rule/route.test.ts

```diff
@@ -189,7 +191,7 @@ describe('Update rule route', () => {
     });
     const defaultAction = getResponseAction();
 
-    test('is successful', async () => {
+    test('is successful for query rule', async () => {
```

Comment:
It could be transformed into `test.each()` with rule params generated via `getCreateRulesSchemaMock ()`, `getCreateEsqlRulesSchemaMock()` etc and provided as a parameter.